Documentation/Contributing/development-flow.md

```diff
@@ -196,6 +196,58 @@ Rebasing is a very powerful feature of Git. You need to understand how it works
 * If there are merge conflicts, you will be prompted to fix them before continuing. Read the output closely. It will tell you how to complete the rebase.
 * When done rebasing, you will see all of your commits in the history.
 
+### Testing changes and debugging issues in Ceph manager modules
+
+The Ceph manager modules are written in Python and can be individually and dynamically loaded from the manager. We can take advantage of this feature in order to make easier to test changes and to debug issues linked with this kind of module.
+This is just a "trick way" to test/debug quickly and easily any modification in manager modules, but do not replace the right way to do the modification, which is the build of a new Ceph container including your modifications.
+
+The ceph dashboard and the rook orchestrator modules are probably the two modules more prone to have modifications affecting the rook cluster, and therefore it is interesting to know how to debug and test changes easily in these modules.
+
+The way to proceed is to make the modification directly in the manager module and reload it:
+
+1. Find out what is your active manager pod and shell into it.
+Using the toolbox:
+```console
+ceph -s
+   cluster:
+   id:     3b140086-e795-4f26-9018-1d234d7f7c98
+   health: HEALTH_OK
+   services:
+     mon: 3 daemons, quorum a,b,c (age 34m)
+     mgr: b(active, since 6h), standbys: a
+```
+
+Exit the toolbox and Shell into the active manager container ( in this example, the mgr 'b' instance): 
+```console
+kubectl exec -n rook-ceph --stdin --tty $(kubectl get pod -n rook-ceph -l ceph_daemon_type=mgr,instance=b  -o jsonpath='{.items[0].metadata.name}') -c mgr  -- /bin/bash
+```
+
+2. Make the modifications needed in the required manager module.
+
+The source code of all the manager modules is in:
+```console
+/usr/share/ceph/mgr/
+```
+
+Enter the folder of the manager module to modify/test and make your changes.
+
+!!! Note
```

Comment:
I don't know if the capitalization matters, but just for consistency with other notes
```suggestion
!!! note
```

Reply:
All the "Note tags" in the document are like "Note". Unfortunately I have seen that the tags "!!! tip" and "!!! Note" we have in the document does not have any "visual" effect when the doc is rendered by github.
I have searched for a solution for remark a note or a tip, but without success.